src/views/ipv/page/pyi-confirm-address.njk

```diff
@@ -46,11 +46,11 @@
             },
             items: [
                     {
-                    value: "next",
+                    value: "fraud",
```

Comment:
I'm not sure I _love_ `fraud` as the event here - since it describes what you think the next state will be (which is the concern of the journey map), rather than the event itself.

I guess better events might be more like `address-current` and `address-changed` or something.

This will be easier once we do the refactoring we discussed, and we can be a bit more flexible about our event names.

Reply:
Yep, that makes sense.

Annoyingly the "next" event has to be a "next" event, as the Address and Fraud CRI only accepts certain input journey events into the nested subjourney.

But address-current is much better than fraud, so that will be clearer at least